specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-03-31/consumption.json

```diff
@@ -1419,7 +1419,7 @@
           "readOnly": true
         },
         "meterId": {
-          "description": "The meter id.",
+          "description": "The meter id (GUID).",
```

Comment:
Sorry I didn't notice this earlier! Adding a hint to the description is an improvement, but I actually mean that you could add a "format" portion of this definition, set to "uuid".

As [the relevant part of the Swagger spec says](https://swagger.io/specification/#dataTypeFormat):
>  the format property is an open string-valued property, and can have any value. Formats such as "email", "uuid", and so on, MAY be used even though undefined by this specification. 


Which would mean this is what these blocks would look like:

``` JSON
"meterId":{
    "description": "The meter id (GUID).",
    "type":"string",
    "format":"uuid",
    "readOnly":true
},
```

Reply:
Sure! I have incorporated this review comment as well.